subprojects/docs/docs.gradle

```diff
@@ -88,7 +88,8 @@ configurations {
 }
 
 dependencies {
-    asciidoctor "org.gradle:docs-asciidoctor-extensions:0.4.0"
+    asciidoctor("org.gradle:docs-asciidoctor-extensions:0.4.0")
+    asciidoctor("org.jruby:jruby-complete:9.2.5.0")
```

Comment:
This should be a constraint instead of a dependency (inside the `constraints {}` block)